Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,65 @@
+public class Game {
+	var word: String
+	let guessManager: GuessManager
+	var view: View
+	public var gameOver: Bool
+	public var winner: Bool
```

Comment:
There's probably a way to avoid holding on to the state of `winner`, it would take some refactoring, but we can wait until friday to pair on it.